content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -892,7 +892,7 @@ prisma db seed
 
 <Admonition type="info">
 
-The `db execute` command is generally available in versions 3.13.0 and later. It is available in preview in versions 3.9.0 and later using the `--preview-feature` CLI flag.
+The `db execute` command is Generally Available in versions 3.13.0 and later. If you're using a version before 3.13.0 and later than 3.9.0, it is available behind a `--preview-feature` CLI flag.
```

Comment:
Maybe simpler? Not sure.
```suggestion
The `db execute` command is Generally Available in versions 3.13.0 and later. If you're using a version between 3.9.0 and 3.13.0, it is available behind a `--preview-feature` CLI flag.
```

Reply:
It works. I thought of this and found it confusing whether 3.13.0 may be perceived as being part of Preview. It's a minor thing.

Reply:
I'll include this. Let's not overthink what this means 😄